staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile/passwordfile.go

```diff
@@ -81,6 +82,7 @@ func NewCSV(path string) (*PasswordAuthenticator, error) {
 	return &PasswordAuthenticator{users}, nil
 }
 
+// AuthenticatePassword returns user info if authentication is successful, nil otherwise
```

Comment:
Document bool response as well? (error is more obvious, although, curiously, I don't see error ever returned in the implementation. Does this implement an interface?).

Reply:
it does implement an interface from `staging/src/k8s.io/apiserver/pkg/authentication/authenticator/interfaces.go`. Strangely `bool` is not documented there as well.